ci/scripts/r_deps.sh

```diff
@@ -41,6 +41,13 @@ fi
 # but we want to error/fail the build.
 # options(warn=2) turns warnings into errors
 ${R_BIN} -e "options(warn=2); install.packages('remotes'); remotes::install_cran(c('glue', 'rcmdcheck', 'sys')); remotes::install_deps(INSTALL_opts = '"${INSTALL_ARGS}"')"
+
+# (Temporarily) install DuckDB from source to avoid their Unity builds (though only if we haven't filtered it out of the deps above)
```

Comment:
```suggestion
# (Temporarily) install DuckDB from source to avoid their Unity builds
# (though only if we haven't filtered it out of the deps above,
# and if we can't get a binary from RSPM)
```

Explaining the `if` in the R code. I guess we don't have any mac/windows builds that use this script (in which case RSPM wouldn't matter, you'd still get a binary)?

Reply:
Yeah, there are no mac or windows builds that use these scripts